src/main/java/org/owasp/wrongsecrets/challenges/Challenge.java

```diff
@@ -17,15 +17,9 @@ public abstract class Challenge {
 
     private final ScoreCard scoreCard;
 
-    public abstract Spoiler spoiler();
+    public abstract Spoiler spoiler() throws Exception;
 
-    /**
-     * method that needs to be overwritten by the Challenge implementation class to do the actual
-     * evaluation of the answer
-     * @param answer String provided by the user
-     * @return true if answer is Correct
-     */
-    protected abstract boolean answerCorrect(String answer);
+    protected abstract boolean answerCorrect(String answer) throws Exception;
```

Comment:
please don't override the global challenge: let's catch any exception in your Challenge29 :)

Reply:
ok then I'll encapsulate them in try and catch statements then